src/controllers/signOut.controller.js

```diff
@@ -0,0 +1,6 @@
+exports.signOut = async function (req, res, next) {
+  res.clearCookie("accessToken");
+  res.clearCookie("refreshToken");
+
+  res.send({ result: "ok", message: "logout! successfully" });
```

Comment:
사소하긴 한데 위 login successful은 ful 로 끝나고, 느낌표가 가장 뒤에 있어서 형식 통일하면 좋을 것 같습니다 !
```suggestion
  res.send({ result: "ok", message: "logout successful!" });
```

Reply:
제가 꼼꼼하지 못했네요! 수정하겠습니다!